jbpm-services/jbpm-kie-services/src/main/java/org/jbpm/kie/services/impl/query/mapper/UserTaskInstanceQueryMapper.java

```diff
@@ -80,7 +80,11 @@ protected UserTaskInstanceDesc buildInstance(DataSet dataSetResult, int index) {
                 getColumnDateValue(dataSetResult, COLUMN_DUEDATE, index),//dueDate
                 getColumnLongValue(dataSetResult, COLUMN_WORKITEMID, index), //workItemId
                 getColumnStringValue(dataSetResult, COLUMN_FORM_NAME, index),
-                getColumnStringValue(dataSetResult, COLUMN_SUBJECT, index)
+                getColumnStringValue(dataSetResult, COLUMN_SUBJECT, index),
+                getColumnStringValue(dataSetResult, COLUMN_CORRELATIONKEY, index), 
+                getColumnIntValue(dataSetResult, COLUMN_PROCESSTYPE, index),
+                getColumnDateValue(dataSetResult, COLUMN_SLA_DUE_DATE, index), 
+                getColumnIntValue(dataSetResult, COLUMN_SLA_COMPLIANCE, index)
```

Comment:
SLA-related columns are used for node(task)-related SLA (from NodeInstanceLog), not for process SLA (from ProcessInstanceLog). These fields are used for the `RuntimeDataServiceImpl#getTaskById(Long taskId, boolean withSLA)` method where you can specify if you want just the task or also its SLA data. So by default it's fine that the mapper doesn't have this data.

At first, the join with NodeInstanceLog was implemented in https://github.com/kiegroup/jbpm/commit/3c4676575601921353e69b41e26c2fc61fd1cedc#diff-031cb54fb8b11dac505ec36874a81a26 but then we found that it negatively impacts performance so we decided to create a separate method for that and reuse the same `UserTaskInstanceDesc` object for that. That's why SLA information is not in the mapper by default.

Reply:
Ok, adding two separate set of contanst for these two columns on task table, in case they are retrieved in future